code/UI/General/SpectatingHint/SpectatingHint.cs

```diff
@@ -8,12 +8,25 @@ public class SpectatingHint : Panel
 {
 	private Panel SwapPanel { get; set; }
 	private Label PlayerName { get; set; }
+	private Label JumpAction { get; set; }
+	private InputGlyph JumpGlyph { get; set; }
 
 	public override void Tick()
 	{
 		if ( Local.Pawn is not Player player )
 			return;
 
+		if ( player.Components.TryGet<PropPossession>( out _ ) )
+		{
+			JumpGlyph.SetButton( InputButton.Duck );
```

Comment:
It's no longer a `JumpGylph` when you set it to `Duck`. We should give it a different name.